client/uploadable.go

```diff
@@ -44,7 +44,7 @@ const (
 // UploadableConfig contains configuration for the AutoUploadable feature
 type UploadableConfig struct {
 	Name    string   `json:"name,omitempty" def:"{name}" descr:"Name for the {feature} feature.\nShould conform to https://docs.bosch-iot-suite.com/things/basic-concepts/namespace-thing-feature/#characters-allowed-in-a-feature-id"`
-	Context string   `json:"context,omitempty" def:"edge" descr:"ID of the {feature} feature."`
+	Context string   `json:"context,omitempty" def:"edge" descr:"Context of the {feature} feature."`
```

Comment:
Is it possible to add some more details? Some references:
* https://github.com/eclipse/vorto/blob/development/models/com.bosch.iot.suite.manager.upload-Uploadable-1.0.0.fbmodel/model.functionblock
* https://websites.eclipseprojects.io/kanto/docs/references/file-upload-config/